app/models/insights_status.rb

```diff
@@ -0,0 +1,59 @@
+class InsightsStatus < HostStatus::Status
+  REPORT_INTERVAL  = 48.hours
+  REGISTER_DATA    = 0 # Should be registered & getting data
+  REGISTER_NO_DATA = 1 # Should be registered, but no data
+  NO_REGISTER      = 2 # Should not be registered & no data
+  NO_REGISTER_DATA = 3 # Should not be registered but getting data
+
+  def self.status_name
+    N_('Insights')
+  end
+
+  def to_label(_options = {})
+    case status
+      when REGISTER_DATA
+        N_('Recieving data')
+      when REGISTER_NO_DATA
+        N_('No data')
+      when NO_REGISTER
+        N_('Not required')
+      when NO_REGISTER_DATA
+        N_('Not installed but recieving data')
```

Comment:
I think a better term would be "Not managed". Insights-client is obviously installed on the host.